libs/sdk-ui-ext/src/internal/dashboardEmbedding/KpiContent/KpiPop.tsx

```diff
@@ -120,12 +144,16 @@ class KpiPop extends PureComponent<IKpiPopProps & WrappedComponentProps> {
     }
 
     getKpiSectionClassName() {
-        const { kpiWidth } = this.props;
+        const { kpiWidth, enableCompactSize } = this.props;
         const kpiSectionItemNode = this.kpiSectionItemNode.current;
-        const className = "gd-flex-container kpi-pop-section";
+        const className = "gd-flex-container headline-compare-section";
+
+        if (enableCompactSize && getResponsiveClassName(kpiWidth)) {
+            return `${className} gd-${getResponsiveClassName(kpiWidth)}`;
+        }
 
         if (kpiSectionItemNode && getResponsiveClassName(kpiWidth)) {
-            return `${className} kpi-pop-section-${getResponsiveClassName(kpiWidth)}`;
+            return `${className} gd-${getResponsiveClassName(kpiWidth)}`;
```

Comment:
This `return` is the same as in the previous `if`, this looks like an error.

Reply:
ye forgot to remove it 😓  was for testing purposes